lib/global.js

```diff
@@ -11,17 +9,25 @@ global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
     return url.indexOf('wss://') === 0 || url.indexOf('ws://') === 0
   })
 
-process.on('SIGINT', ensureCloseElectron)
-process.on('SIGTERM', ensureCloseElectron)
+global.WRTC = function () {
+  var wrtc = ElectronWebRTC()
+  
+  global.WRTC = wrtc
```

Comment:
Can you remove this line? I don't think this is necessary. We only use `global.WRTC` in one place in `webtorrent`, and you already changed it to handle `global.WRTC` being a function. It's odd for it to change into an object after being called, IMO.